app/scripts/templates/connect_another_device.mustache

```diff
@@ -23,7 +23,7 @@
 
       <form novalidate>
         <div class="button-row">
-          <a data-flow-event="link.signin" id="signin" class="button primary-button" href="{{{ escapedSignInUrl }}}">{{#unsafeTranslate}}Sign in &rarr;{{/unsafeTranslate}}</a>
+          <a data-flow-event="link.signin" id="signin" class="button primary-button" href="{{{ escapedSignInUrl }}}">{{#unsafeTranslate}}Sign in {{/unsafeTranslate}}</a>
```

Comment:
`unsafeTranslate` is used to display strings that contain HTML, since the `&rarr` is removed the string no longer contains any HTML. Can you change `unsafeTranslate` to `t`? Can you also remove the extra space after `in` before the `{`?